spec/client_spec.rb

```diff
@@ -59,8 +52,12 @@
   end
 
   context 'get' do
-    it 'returns array on success' do
-      expect(client.get_posts).to be_an(Array)
+    it 'returns Takeout::Response on success' do
+      expect(client.get_posts).to be_an(Takeout::Response)
+    end
+
+    it 'returns an array in its body on success' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.